clustered/server/src/main/java/org/ehcache/clustered/server/offheap/OffHeapServerStore.java

```diff
@@ -29,14 +32,29 @@
 public class OffHeapServerStore implements ServerStore {
 
   private final List<OffHeapChainMap<Long>> segments;
+  private final List<ServerStoreEvictionListener> listeners = new CopyOnWriteArrayList<ServerStoreEvictionListener>();
```

Comment:
You're right, there will never be a need for multiple listeners. Let me rationalize all that.